src/Controller/Project/CcaController.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace App\Controller\Project;
+
+use App\Entity\Project\Cca;
+use App\Entity\Project\Ce;
+use App\Form\Project\CcaType;
+use App\Form\Project\SubCcaType;
+use App\Service\Project\DocTypeManager;
+use DateTime;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CcaController extends AbstractController
+{
+    /**
+     * @Security("has_role('ROLE_SUIVEUR')")
+     * @Route("/suivi/cca", name="project_cca_index")
+     *
+     * @return RedirectResponse|Response
+     */
+    public function index(): Response
+    {
+        $em = $this->getDoctrine()->getManager();
```

Comment:
C'est mieux d'utiliser l'injection de dépendance:

```suggestion
    public function index(EntityManagerInterface $em): Response
    {
```